connector_routific/security/res_groups.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2021 Tecnativa - Carlos Roca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).-->
+<odoo>
+    <record id="group_routific_manager" model="res.groups">
```

Comment:
Why creating a specific group for this and not binding it to an existing one? And now you will need `sudo` in all the operations that needs this data.